projects/plugins/jetpack/modules/stats.php

```diff
@@ -821,7 +821,7 @@ function stats_admin_bar_menu( &$wp_admin_bar ) {
 	$menu = array(
 		'id'    => 'stats',
 		'href'  => add_query_arg( 'page', 'stats', admin_url( 'admin.php' ) ), // no menu_page_url() blog-side.
-		'title' => "<div><img src='$img_src' srcset='$img_src 1x, $img_src_2x 2x' width='112' height='24' alt='$alt' title='$title'></div>",
+		'title' => "<div><img fetchPriority='low' loading='lazy' decoding='async' src='$img_src' srcset='$img_src 1x, $img_src_2x 2x' width='112' height='24' alt='$alt' title='$title'></div>",
```

Comment:
I guess we probably would want to use all lower case for `featchPriority` as it's an HTML property?

Reply:
good catch!